### PR TITLE
Add RH Cloud manual tests based on BZ scenarios 

### PR DESCRIPTION
**Pull request by jameerpathan111**

### CODE REVIEW

**Review by omkarkhatavkar — APPROVED** [review]

File: tests/foreman/cli/test_rhcloud_inventory.py [hunk 1]
```diff
@@ -139,3 +132,79 @@ def test_positive_sync_inventory_status():
 
     :CaseLevel: System
     """
+
+
+@pytest.mark.stubbed
+def test_rhcloud_inventory_bz_1962694():
```
Comment (mshriver) [inline tests/foreman/cli/test_rhcloud_inventory.py:138]: I don't think naming the test case for the BZ is a good practice. The BZ context is available in the metadata, and provides zero context when viewing test results concerning what the test function is for. Can't tell a simple field validation test from a complex integration test.

Please name the test cases according to their relevant coverage. 

Is there no way to automate any of these three test cases, as indicated by `ManualOnly` metadata?

File: tests/foreman/cli/test_rhcloud_inventory.py [hunk 2]
```diff
@@ -139,3 +132,79 @@ def test_positive_sync_inventory_status():
...
+            ForemanInventoryUpload::Async::GenerateAllReportsJob.perform_now
+
+    :expectedresults: If organization had more hosts than specified by max_org_size variable
+        then report won't be uploaded.
+
+    :CaseImportance: Low
+
+    :BZ: 1962694
+
+    :CaseAutomation: ManualOnly
```
Comment (mshriver) [inline tests/foreman/cli/test_rhcloud_inventory.py:158]: Is there no way to automate any of these test cases, as indicated by `ManualOnly`?

**Review by mshriver — CHANGES_REQUESTED** [review]
Please update test names

File: tests/foreman/cli/test_rhcloud_inventory.py (hunk 2 above)
Comment (jameerpathan111) [reply tests/foreman/cli/test_rhcloud_inventory.py:158]: They are hard to automate, some are destructive in nature and not worth automating.

**Comment by jameerpathan111** [comment]
> Please update test names

@mshriver I have updated the test names.

File: tests/foreman/cli/test_rhcloud_inventory.py (hunk 1 above)
Comment (jameerpathan111) [reply tests/foreman/cli/test_rhcloud_inventory.py]: @mshriver updated the pr.

File: tests/foreman/cli/test_rhcloud_inventory.py (hunk 2 above)
Comment (mshriver) [reply tests/foreman/cli/test_rhcloud_inventory.py:158]: Ack, double checking this is what you wanted, instead of `NotAutomated`

**Review by mshriver — APPROVED** [review]

**Pull request merged**